sdcm/tester.py

```diff
@@ -201,7 +201,11 @@ def _store_test_result(parent, exc_val, exc_tb, name):
 
 
 def critical_failure_handler(signum, frame):  # pylint: disable=unused-argument
-    raise AssertionError("Critical Error has failed the test")
+    try:
+        if TestConfig().tester_obj().teardown_started:
+            return
```

Comment:
Maybe add here warning log message to be able to see how often we face it?

Reply:
Thanks, added